index.js

```diff
@@ -1173,7 +1175,7 @@
     }
   }
 
-  //# test :: (Array Type, Type, a) -> Boolean
+  //# test :: Array Type -> Type -> a -> Boolean
```

Comment:
why have changed the signature to curried form? looks like the `test` function is not curried